src/coreclr/scripts/superpmi.py

```diff
@@ -2565,7 +2659,7 @@ def verify_replay_common_args():
 
         # Make a more intelligent decision about the arch and build type
         # based on the path of the jit passed
-        if standard_location and not coreclr_args.build_type in coreclr_args.jit_path:
+        if jit_in_product_location and not coreclr_args.build_type in coreclr_args.jit_path:
```

Comment:
a nit: one operator versus two
```suggestion
        if jit_in_product_location and coreclr_args.build_type not in coreclr_args.jit_path:
```